package.json

```diff
@@ -109,7 +108,7 @@
     "favicons": "5.3.0"
   },
   "engines": {
-    "node": ">= 10.*"
+    "node": "10.* || 12.* || >= 14"
```

Comment:
Drop Node.js 13 as it's end of life

Reply:
Just to call out, this is the main breaking change here.